src/mlpack/tests/neat_test.cpp

```diff
@@ -44,27 +44,27 @@ BOOST_AUTO_TEST_SUITE(NEATTest)
 /**
  * Test NEAT on the XOR Test.
  */
-BOOST_AUTO_TEST_CASE(NEATXORTest)
-{
-  arma::arma_rng::set_seed_random();
-  XORTask task;
-  NEAT<XORTask, HardSigmoidFunction, RankSelection> model(task, 2, 1, 100, 150, 10);
-  model.FinalFitness() = 3.8;
+// BOOST_AUTO_TEST_CASE(NEATXORTest)
```

Comment:
I think as a quick example we should leave the XOR test as well.

Reply:
Yup I'll leave it in, but do you think I should also write a short guide?